functions/Get-DbaUptime.ps1

```diff
@@ -134,7 +134,7 @@ function Get-DbaUptime {
 				Select-DefaultView -InputObject $rtn -ExcludeProperty WindowsBootTime, WindowsUptime, SinceWindowsBoot
 			}
 			else {
-				Select-DefaultView -InputObject $rtn -Property ($rtn|get-member -MemberType NoteProperty|select-object -ExpandProperty name)
```

Comment:
You can remove the whole if/else block since we dropped the parameters for the output.

Remove the variable `$rtn` and just have the custom object output straight. 
```powershell
			[PSCustomObject]@{
				ComputerName     = $WindowsServerName
				InstanceName     = $server.ServiceName
				SqlServer        = $server.Name
				SqlUptime        = $SQLUptime
				WindowsUptime    = $WindowsUptime
				SqlStartTime     = $SQLStartTime
				WindowsBootTime  = $WinBootTime
				SinceSqlStart    = $SQLUptimeString
				SinceWindowsBoot = $WindowsUptimeString
			}
```